src/components/AppSettingsMenu.vue

```diff
@@ -71,6 +66,10 @@ export default {
 	data() {
 		return {
 			loadingAvatarSettings: false,
+			text: t(
```

Comment:
Would it be possible not to wrap the line here or was that done by the linter?

Reply:
yes, i ran the linter, so its linters fault